app/Actions/Place/SpecialFeature/CreatePlaceFeatureAction.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Hedonist\Actions\Place\SpecialFeature;
+
+
+use Hedonist\Repositories\Place\PlaceFeatureRepositoryInterface;
+use Hedonist\Entities\Place\PlaceFeature;
+
+
+class CreatePlaceFeatureAction
+{
+    protected $repository;
+
+    /**
```

Comment:
Remove all php docs it is not necessary. It has a sense if you use a variable implicitly

Reply:
done